frontend/src/components/table/body/TableBodyCell.tsx

```diff
@@ -40,7 +40,7 @@ export const TableBodyCell = ({
           >
             {cell.value}
           </Link>
-          {status === Status.NEW && (
+          {status && status === Status.NEW && (
```

Comment:
`status === Status.New` already excludes the case of `!status`, right? Any need for this change?

Reply:
I was getting an error if status wasn't being passed in...